campgns/keeporig/map00020.txt

```diff
@@ -174,6 +181,18 @@ ENDIF
 REM Optional: taking down PLAYER1***********************************************
 REM ****************************************************************************
 
+IF(PLAYER1,GAME_TURN > 122500)
```

Comment:
maybe add a comment for how long 122500 game turns is

Reply:
done